app/publishing/publish-statement.js

```diff
@@ -24,13 +25,20 @@ const publishStatement = async (request) => {
   try {
     validateEmail(request.email)
     const personalisation = getPersonalisation(request.scheme.name, request.scheme.shortName, request.scheme.year, request.scheme.frequency, request.businessName, request.paymentPeriod)
+    //TODO Add check if email errored out
```

Comment:
We catching errors here but will also need to check the Notify API for status updates periodically.